.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Bug'
+description: "Report something that doesn't work how it should. Whenever possible, please open a ticket with [GCP support](https://cloud.google.com/support-hub) instead."
+labels: ['bug']
+body:
+- type: checkboxes
+  id: checklist
+  attributes:
+    label: Checklist
+    options:
+    - label: I did not find a related open issue.
+      required: true
+    - label: 'I did not find a solution in the troubleshooting guide: (https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/troubleshooting-config-sync)'
+      required: true
+    - label: 'If this issue is time-sensitive, I have submitted a corresponding issue with [GCP support](https://cloud.google.com/support-hub).'
+      required: true
+    - label: 'I have attached a [nomos bugreport](https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/nomos-command#bug) to this issue.'
+      required: true
+- type: 'markdown'
+  attributes:
+    value: |-
+      Thank you for filing a bug. Please complete the form below so we can triage, reproduce, and fix your issue.
+
+- id: 'bug-desc'
+  type: 'textarea'
+  attributes:
+    label: 'Bug Description'
+    description: 'A clear and concise description of the bug.'
+  validations:
+    required: true
+
+- id: 'optional-diag'
+  type: 'textarea'
+  attributes:
+    label: 'Additional Diagnostic Information'
+    description: 'Any additional diagnostic info'
+  validations:
+    required: true
+
+- type: input
```

Comment:
Suggest adding a section for the enviroment (i.e. the cluster type, standard / autopilot or public / private) for faster turn around.

Reply:
Oh I see the type in the description, so maybe update the name to reflect that.

Reply:
updated